runtime/assembly/modules/storage.ts

```diff
@@ -16,19 +16,23 @@ export namespace Storage {
         const key64: u64 = Serialiser.serialise_result(Utils.toU8Array(u8Key));
         const result: u64 = ext_storage_get_version_1(key64);
         const ptr_size: i32[] = Serialiser.separate_pointer_size(result);
-        return Serialiser.deserialise_input(ptr_size[0], ptr_size[1]);
+        let valueu8: u8[] = Serialiser.deserialise_input(ptr_size[0], ptr_size[1]);
+        if (valueu8[0] == 1){
+            return valueu8.slice(2, valueu8.length);
+        }
+        return valueu8;
     }
 
     /**
-    * Sets the value in storage for a given key
-    * @param pair an array with key, value pair
-    */
+     * Sets the value in storage for a given key
+     * @param key key pair
+     * @param value value of the pair
+     */
     export function set(key: string, value: string): void {
```

Comment:
Same comment for the `set` -> I think that it will be better to use `u8[]` instead of `string` for both `key` and `values`